nextjs/src/app/(marketing)/layout.tsx

```diff
@@ -17,11 +18,11 @@ export default async function Layout({
   return (
     <div className='flex flex-col min-h-dvh'>
       <AreaPattern />
-      <Navbar isLoggedIn={isLoggedIn} />
       <main className="p-4 relative">
+      <FeedbackBanner/>
+      <Navbar isLoggedIn={isLoggedIn} />
```

Comment:
question: should the <nav> element be inside <main>?

Reply:
No, I'll fix that